2-Browsers/Week1/homework/ex4-whatsTheTime/index.js

```diff
@@ -8,7 +8,24 @@ Full description at: https://github.com/HackYourFuture/Homework/tree/main/2-Brow
 2. Have the function execute when it's loading in the browser.
 ------------------------------------------------------------------------------*/
 function addCurrentTime() {
-  // TODO complete this function
+  let date = new Date(); 
+  let hours = date.getHours();
+  let minutes = date.getMinutes();
+  let seconds = date.getSeconds();
+
+  hours = (hours < 10) ? "0" + hours : hours;
+  minutes = (minutes < 10) ? "0" + minutes : minutes;
+  seconds = (seconds < 10) ? "0" + seconds : seconds;
+    
+  let time = hours + ":" + minutes + ":" + seconds;
+  document.getElementById("clock").textContent = time; 
+  setTimeout(function(){addCurrentTime()}, 1000);
```

Comment:
If you want to pass a function as an argument to another function you can just write the function but without the ending params. Don't need to wrap it in an anonymous function

```suggestion
  setTimeout(addCurrentTime, 1000);
```